GoogleForCreators/web-stories-wp
# Use attrs for avoiding creation of excess classes by Styled Components

## Summary



## Relevant Technical Choices
- Uses `attrs` in places where otherwise SC would create hundreds of classes.


## To-do

- [x] Fix Karma



## User-facing changes
N/A


## Testing Instructions
Open the Text Sets panel. Scroll to the end of the text sets. Now filter the text sets, scroll to the end, remove the filter and scroll again. Verify that no warning about 200+ classes is displayed in the console.



### QA






This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Checklist




---



Fixes #5797
